public/app/features/plugins/components/PluginsErrorsInfo.tsx

```diff
@@ -1,51 +1,68 @@
 import { css } from '@emotion/css';
 import React from 'react';
 
-import { GrafanaTheme2, PluginErrorCode, PluginSignatureStatus } from '@grafana/data';
+import { GrafanaTheme2, PluginErrorCode, PluginSignatureStatus, PluginType } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
-import { HorizontalGroup, InfoBox, List, PluginSignatureBadge, useStyles2 } from '@grafana/ui';
+import { Alert, HorizontalGroup, Icon, List, PluginSignatureBadge, useStyles2 } from '@grafana/ui';
 
 import { useGetErrors, useFetchStatus } from '../admin/state/hooks';
 
-export function PluginsErrorsInfo() {
-  const errors = useGetErrors();
+type PluginsErrorInfoProps = {
+  filterByPluginType?: PluginType;
+};
+
+export function PluginsErrorsInfo({ filterByPluginType }: PluginsErrorInfoProps) {
+  let errors = useGetErrors();
   const { isLoading } = useFetchStatus();
   const styles = useStyles2(getStyles);
+  if (filterByPluginType) {
+    errors = errors.filter((pluginError) => {
+      const pluginIdParts = pluginError.pluginId.split('-');
```

Comment:
There are generally plugins out there that don't follow the name convention of putting `-datasource` at the end and we risk underexposing possible errors here if we base our filtering in the plugin id.

All the information about a plugin is available but filtered out by the `PluginError` that the `userGetErrors` hook return. 

A better way to do this and be sure you are not hiding possible errors is to modify that selector https://github.com/grafana/grafana/blob/main/public/app/features/plugins/admin/state/selectors.ts#L66 (and its associated type) to also return the plugin type.

Reply:
Ah nice! I'll update the selector in a bit. Thanks @academo 👍

Reply:
Updated in [this commit](https://github.com/grafana/grafana/pull/74339/commits/ef9c96454de20a2de4e525f375719a92bbd2c3af).